pinot-core/src/main/java/org/apache/pinot/core/segment/index/column/ColumnIndexContainer.java

```diff
@@ -42,14 +43,14 @@
   InvertedIndexReader<?> getInvertedIndex();
 
   /**
-   * Returns the inverted index for the column, or {@code null} if it does not exist.
+   * Returns the range index for the column, or {@code null} if it does not exist.
```

Comment:
Are we planning to do similar cleanup for range index as well? It is also based off InvertedIndex (for the same reasons why TextIndex was based)

Reply:
Yes, will do that in a separate PR. We need to design the interface for the range index